docs/experience_dates_search/index.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: default
+title: Experience dates search
+has_children: true
+nav_order: 8
+has_toc: false
+---
+
+# Experience dates search
+
+Experience dates search enables you to filter a Creator's experiences with a separate result for each departure date of each experience.
+
+The experience dates search is executed using the [experience_dates_search tag]({% link docs/reference/tags/experience_dates_search_tag/index.md %}). The tag returns an `items` array of [ExperienceDates]({% link docs/reference/objects/product/experience_date.md %}), a `paginate` [Pagination]({% link docs/reference/objects/pagination.md %}) object and a `search` [Search]({% link docs/reference/objects/search_query.md %}) object. 
```

Comment:
Let's follow the same pattern as `product_search` and keep it singular, i.e. `experience_date_search`